util/verible-format.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import os
+import argparse
+import subprocess
+import logging
+import difflib
+
+logger = logging.getLogger('verible_format')
+
+VERIBLE_ARGS = ["--formal_parameters_indentation=indent",
+                "--named_parameter_indentation=indent",
+                "--named_port_indentation=indent",
+                "--port_declarations_indentation=indent"]
+
+
+def get_repo_top():
+    repo_top = subprocess.Popen('git rev-parse --show-toplevel',
+                                stdout=subprocess.PIPE, shell=True)
+    return repo_top.stdout.read().decode().strip()
```

Comment:
Possibly simpler?
```python
return subprocess.run(['git', 'rev-parse', '--show-toplevel'],
                      check=True, text=True, capture_output=True).stdout.strip()
```
(no need for the shell parsing or the explicit `Popen` call).

Reply:
Like that. Replaced.